tests/integration/kubernetes/k8s-confidential-attestation.bats

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env bats
+# Copyright 2024 IBM Corporation
+# Copyright 2024 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+load "${BATS_TEST_DIRNAME}/../../common.bash"
+load "${BATS_TEST_DIRNAME}/confidential_common.sh"
+load "${BATS_TEST_DIRNAME}/tests_common.sh"
+
+export test_key="aatest"
+export SUPPORTED_TEE_HYPERVISORS=("qemu-sev" "qemu-snp" "qemu-tdx" "qemu-se")
+export SUPPORTED_NON_TEE_HYPERVISORS=("qemu")
+
+setup() {
+	if ! [[ " ${KATA_HYPERVISOR} " =~ " ${SUPPORTED_TEE_HYPERVISORS[@]} " ]] && ! [[ " ${KATA_HYPERVISOR} " =~ " ${SUPPORTED_NON_TEE_HYPERVISORS} " ]]; then
+		skip "Test not supported for ${KATA_HYPERVISOR}"
+	fi
+
+	get_pod_config_dir
+
+	# TODO need to get CC_KBS deployed with sample KBS based on info in
+	# https://github.com/confidential-containers/confidential-containers/issues/184#issuecomment-1894373294
+	# with the resource content
+	# "default/aa/key": $($test_key | base64)
+}
+
+@test "Test we can get evidence from CDH resource" {
+
+	# TODO - do we still need AA_KBC set and checked - do we have any other options?
+	if [ "${AA_KBC}" = "cc_kbc" ]; then
```

Comment:
@wainersm and @fidencio so seems that we still need to do this for TEEs, however, seems like we still also need to do the add_kernel_params if I an not mistaken

Reply:
That's a good question. On CCv0 branch we had the Offline_fs KBC and cc_kbc, I don't know the situation on `main`. If $AA_KBC is required then probably it should be exported from the github workflow.

@fitzthum @stevenhorsman please come to rescue us!

Reply:
For the time being we do need to set `aa_kbc_params` as a kernel param. The value should be `cc_kbc::<uri-of-kbs>`. The KBS URI needs to be accessible from inside the guest. @wainersm I'm not sure what the easiest way to get this value is. Maybe this is something we want to add to the KBS utils.

I'm not sure if we want to have the `AA_KBC` var as input to the test. In the past we used that var almost as a general switch for turning on CoCo stuff, but that was a little bit weird. Here we are really using the param just as the KBS URI, although on some platforms the first part will be different too.

Note that we no longer process this param in the kata agent. There were some comments about this on Linda's PR. We do still need to set it, though. Before too long we will probably replace this with a config in the init-data, but that is not ready yet.

Reply:
@wainersm is that possible to add in the KBS utils?

Reply:
hi @GabyCT , `kbs_k8s_svc_http_addr` should return the `http://host:port` address of KBS. We just need to call that function.

Reply:
@wainersm this is done here a4638c02fb388e18146a55871729f1eb30fdd14d